modules/builtin/pulseaudio.go

```diff
@@ -0,0 +1,110 @@
+package builtin
+
+import (
+	"log"
+
+	"fyne.io/fyne"
+	"fyne.io/fyne/layout"
+	"fyne.io/fyne/theme"
+	"fyne.io/fyne/widget"
+	"github.com/mattboll/pulseaudio"
+
+	"fyne.io/fynedesk"
+	wmtheme "fyne.io/fynedesk/theme"
+)
+
+var pulseaudioMeta = fynedesk.ModuleMetadata{
+	Name:        "Pulseaudio",
+	NewInstance: NewPulseaudio,
+}
+
+// Pulseaudio is a progress bar module to modify screen pulseaudio
```

Comment:
I suspect that "modify screen pulseaudio" is a copy-paste bug :-p

Reply:
/o\ shame on me !